x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -417,23 +420,34 @@ async function _compilePackagePolicyInput(
     throw new Error(`Input template not found, unable to find input type ${input.type}`);
   }
 
-  if (!packageInput.template_path) {
-    return undefined;
-  }
+  let compiledTemplate;
+  if (packageInput.template_path) {
+    const [pkgInputTemplate] = await getAssetsData(registryPkgInfo, (path: string) =>
+      path.endsWith(`/agent/input/${packageInput.template_path!}`)
```

Comment:
Do we need the bangs here and later if we are in an `if` which says it's truthy?